addons/mail/static/src/models/messaging.js

```diff
@@ -176,7 +176,7 @@ registerModel({
                     case 'create':
                         return ormService.create(model, args[0], kwargs);
                     case 'read':
-                        return ormService.read(model, args[0], args.length > 1 ? args[1] : undefined, kwargs);
+                        return ormService.read(model, args[0], args.length > 1 ? args[1] : kwargs.fields, _.omit(kwargs, 'fields'));
```

Comment:
We would like to stop using `_` lib, we could simply use `{ ...kwargs, fields: undefined }` instead or something like that.

Doesn't other cases have the same problem too by the way?

Reply:
Or if the existence of the key is checked instead of `undefined`, `delete kwargs.fields`.

Reply:
> We would like to stop using `_` lib, we could simply use `{ ...kwargs, fields: undefined }` instead or something like that.
> 
> Doesn't other cases have the same problem too by the way?
Even if I do this way, traceback has come so we have to remove it everywhere: 
```
 await this.messaging.rpc({
        model: 'res.users',
        method: 'search',
        args: [[['id', '=', 1]]],
        kwargs: {
              context,
              fields,
        },
     }, { shadow: true });
```

Reply:
Thank you for checking, if you `know` search has the bug too you could fix it too please!

And verify all the other methods so that we sure all of them are fine.